### PR TITLE
docs/executors: uncomment multiqueue 5.1 additions 

### PR DESCRIPTION
**Pull request by sanderginn**
To be merged when 5.1 is released.

## Test plan
Nothing, doc changes

### CODE REVIEW

**Comment by sanderginn** [comment]
Current dependencies on/for this PR:
* main
  * **PR #53966** <a href="https://app.graphite.dev/github/pr/sourcegraph/sourcegraph/53966" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/sourcegraph/sourcegraph/53966?utm_source=stack-comment).

**Comment by sanderginn** [comment]
@sourcegraph/executors feel free to push any additions that should be merged after the release.

File: doc/admin/executors/deploy_executors_kubernetes.md
Comment (Piszmog) [inline doc/admin/executors/deploy_executors_kubernetes.md:1]: #53780 will supersede this change. So maybe revert to avoid conflict?

Comment (sanderginn) [reply doc/admin/executors/deploy_executors_kubernetes.md:1]: > _approves_

> _instantly forgets about these changes_

🤦

Comment (Piszmog) [reply doc/admin/executors/deploy_executors_kubernetes.md:1]: We can keep this open and merge after https://github.com/sourcegraph/sourcegraph/pull/53780 or you can push to that branch

**Review by Piszmog — APPROVED** [review]

**Pull request merged**